### DISCUSSION TITLE
How to integrate a comment system to posts, perhaps like https://disqus.com/

### DISCUSSION THREAD

**Discussion by RoylerMarichal** [austintoddj/canvas#983] [New Features]
I need to add functionality for users to comment on articles

**Comment by austintoddj** [comment]
Since this feature is fairly unique, I don't see it as something that Canvas will include out of the box. If enough people would find it to be useful, then I could reconsider.

That being said, if I were going to add commenting functionality, I'd reach for [Utterances](https://utteranc.es/) or [Disqus](https://disqus.com/). If you're using `canvas-ui`, there'd be some refactoring of the Vue components you would need to do to achieve what you're looking for.

**Comment by austintoddj** [comment]
Closing as obsolete with [Canvas v7.0.0](https://github.com/austintoddj/canvas/releases/tag/v7.0.0).
